src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -685,6 +697,9 @@ internal override Compilation WithEventQueue(AsyncQueue<CompilationEvent>? event
                 eventQueue);
         }
 
+        // Nullable analysis data for methods, parameter default values, and attributes. Collected during testing only.
+        internal ConcurrentDictionary<object, int>? NullableAnalysisData;
+
```

Comment:
Consider placing the field declaration at the beginning of the type declaration. I couldn't figure out why this place was chosen, it doesn't look like there is any relationship to surrounding code.